server/src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/DateHistogramAggregationBuilder.java

```diff
@@ -70,6 +72,7 @@
 public class DateHistogramAggregationBuilder extends ValuesSourceAggregationBuilder<ValuesSource.Numeric, DateHistogramAggregationBuilder>
         implements MultiBucketAggregationBuilder {
     public static final String NAME = "date_histogram";
+    private static DateMathParser DEFAULT_DATE_PARSER = new DateMathParser(DateFieldMapper.DEFAULT_DATE_TIME_FORMATTER);
```

Comment:
Should we use the epoch_millis parser specifically rather than a parser that understands epoch millis among other formats?

Reply:
I thought I did that, might have not pushed it though because I think it was a late addition. Will do.